Daml Finance: release notes adjustments (#552)
digital-asset/docs.daml.com: docs/2.8.0/docs/daml-finance/index.rst

and upgradability. Along with code changes, we have added new documentation and tutorials to
streamline the learning process for new users.

This section outlines the major changes and reasons behind them. The technical changelog for each
package can be found as a sub-page :doc:`here <packages/index>`.

Enhanced Upgradeability, Extensibility, and Interoperability
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> We have enhanced the core asset model (``Account``, ``Holding``, and ``Instrument`` interfaces) to

Reviewer: ```suggestion
The core asset model (``Account``, ``Holding``, and ``Instrument`` interfaces) has been enhanced to
```

streamline upgrade processes, enhance extensibility, and improve interoperability:

#. The ``Account`` now links to its ``Daml.Finance.Interface.Holding.Factory`` by key, a
   ``HoldingFactoryKey``, instead of a ``ContractId``. This facilitates the upgrading of a
   ``HoldingFactory`` without the need to modify existing ``Account`` contract instances. It also

>    enables a "lazy" upgrade approach for holdings, as detailed in our new

Reviewer: ```suggestion
   enables a "lazy" upgrade approach for holdings, as detailed the
```

   :doc:`Holding Upgrade Tutorial <./tutorials/upgrade/holding>`.

#. In anticipation of the need for standardization when implementing composed workflows across

>    applications, we have introduced the notion of a ``HoldingStandard`` (as part of the

Reviewer: ```suggestion
   applications, there is the notion of a ``HoldingStandard`` (as part of the
```

Author: changed to:

In anticipation of the need for standardization when implementing composed workflows across
applications, the notion of a ``HoldingStandard`` was introduced (as part of the ``InstrumenKey``).

   ``InstrumenKey``). It categorizes holdings into four distinct classes, each defined by the
   combination of holding interfaces (``Transferable``, ``Fungible``, and ``Holding``) they
   implement. This new standard has guided the renaming and structuring of holding implementations.
   The ``Fungible`` interface no longer requires the ``Transferable`` interface. However, both
   ``Transferable`` and ``Fungible`` continue to require the implementation of the ``Holding``
   interface (renamed from ``Base`` following customer feedback). Moreover, the settlement process
   has been refined to require only a matching ``HoldingStandard``, allowing for implementation
   variations.

#. A unified ``HoldingFactory`` capable of creating holdings for any specified ``HoldingStandard``
   has been adopted. In particular, this enables multiple holdings (of various ``HoldingStandards``)
   to be credited to the same account.

Foreseeing future integration with Daml 3.0 and the Canton Network
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In order to ease future transitions to Daml 3.0 and the Canton Network we have shifted to

Reviewer: ```suggestion
In order to ease future transitions to Daml 3.0 and the Canton Network, there is a shift to
```

single-maintainer contract keys:

#. The `issuer : Party` of the ``InstrumentKey`` is now the single maintainer for the ``Instrument``
   key.

#. For ``Batch`` and ``Instruction``, the `requestors : Parties` field has been divided into a
   single-maintainer `instructor : Party` for the ``Instruction`` key, alongside additional
   signatories `consenters : Parties`. Corresponding changes have been made to the ``Batch`` and
   ``Instruction`` views. In the ``Daml.Finance.Lifecycle.Rule.Claim`` implementation,
   `providers : Parties` has been replaced with a single `provider : Party` (to facilitate assigning

>    the `provider` as a settlemetn `instructor : Party`).

Reviewer: ```suggestion
   the `provider` as a settlement `instructor : Party`).
```

> #. The LedgerTime key has been completely removed as it was redundant.

Reviewer: ```suggestion
#. The LedgerTime key has been completely removed, as it was redundant.
```

interfaces with their related ``Reference`` contract instance, a ``Remove`` choice has been added to
the ``Account``, base ``Instrument``, and ``HoldingFactory`` interfaces.

New Interface Lockable
~~~~~~~~~~~~~~~~~~~~~~

The locking mechanism has been separated from the base ``Holding`` interface into a new ``Lockable``
interface (which the ``Holding`` now requires). That makes ``Lockable`` available for broader use;
while the ``Account`` also implements ``Lockable`` allowing to freeze an account, it’s not required.

Additionally, the implementations of ``Transfer``, ``Split``, ``Merge``, and ``Debit`` have been
adjusted to require unlocking before they can be used when in the locked state.

New Instruments
~~~~~~~~~~~~~~~

> Furthermore, we have broadened the library's functionality by introducing new financial instruments,

Reviewer: ```suggestion
The library's functionality has been broadened by introducing new financial instruments,
```

> such as structured products, and multi-underlying asset swap instruments (both early access).

Reviewer: ```suggestion
such as structured products and multi-underlying asset swap instruments (both early access).
```

Usability Improvements
~~~~~~~~~~~~~~~~~~~~~~

Finally, a large number (around 50 tickets) of smaller improvements addressing customer feedback

> have been made. These improvements range from the consistency of naming conventions (e.g. the type

Reviewer: ```suggestion
were made. These improvements range from the consistency of naming conventions (for example, the type
```
